source/includes/changelogs/releases/100.9.1.rst

```diff
@@ -5,12 +5,20 @@
 
 *Released 2023-11-09*
 
-This version updates external dependencies and fixes an issue where
-mongodump could change the ordering of fields in a view definition.
+This version updates external dependencies and fixes an issue where mongodump 
+could change the ordering of fields in a view definition. View pipelines and 
```

Comment:
Could we just add a link to the ticket in the description? Something like this?

```
This version updates external dependencies and fixes an issue where mongodump 
could change the ordering of fields in a view definition (:issue:`TOOLS-3411`).
```

Reply:
Sure thing!